src/utils/border.scss

```diff
@@ -0,0 +1,36 @@
+$borders: (
+  'Right': right,
+  'Left': left,
+  'Bottom': bottom,
+  'Top': top
+);
+
+@each $name, $value in $borders {
+  .u-border#{$name} {
+    border-#{$value}: 1px solid $cu-divider--light;
+  }
+  .u-border#{$name}None {
+    border-#{$value}: 0;
+  }
+  .u-borderNone {
+    border: 0;
+  }
+}
+
+@if $responsive == true {
```

Comment:
Since the border logic is now being handled within the `Panel--stacked` class, the responsive utilities aren't really necessary yet, but if they seem useful we can keep them in.

Reply:
I think there's use in keeping them